ansible/roles/sonarqube/tasks/configure_linux.yml

```diff
@@ -21,7 +27,6 @@
     group: "{{ sonar_group }}"
     shell: "/sbin/nologin"
     comment: "SonarQube System user"
-    createhome: no
```

Comment:
We should probably make the user and group system users and groups. As per [this](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/group_module.html#parameter-system)

Reply:
Group and User now both updated to system